schemas/login.schema.js

```diff
@@ -144,6 +144,36 @@ const childSchema = {
   },
 }
 
+const bankruptcySchema = {
+  dobDay: {
+    ...validationArray([isValidDay]),
+  },
+  dobMonth: {
+    isInt: {
+      errorMessage: 'errors.login.dateOfBirth.validMonth',
+      options: { min: 1, max: 12 },
+    },
+  },
+  dobYear: {
+    isInt: {
+      errorMessage: 'errors.login.dateOfBirth.validYear',
+      options: { min: currentDate.getFullYear() - 200, max: currentDate.getFullYear() - 1 },
+    },
+  },
+  trusteeFirstName: {
+    isEmpty: {
+      errorMessage: 'errors.login.trusteeFirst',
+      negated: true,
+    },
+  },
```

Comment:
What if we don't make first name mandatory? Let's just remove this until we hear otherwise.

Reply:
(the reason is so that it will be in line with the "your child" validation.)